docs/getting_started.md

```diff
@@ -0,0 +1,129 @@
+
+3. [Creating your first APB](#first-apb)
+    1. [Init](#init)
+    2. [Actions](#actions)
+        1. [Provision](#provision)
+        2. [Bind](#bind)
+        3. [Unbind](#unbind)
+        4. [Deprovision](#deprovision)
+        
+4. [More information](#more-info)
+
+<a name="intro"></a>
+## Introduction Ansible Playbook Bundles (APBs)  
+ 
+In this tutorial, we'll walk through the creation of a APB for a PostgreSQL Database.  You will be able to provision, deprovision, and bind or unbind it to other applications.  You can find more information about the design of APBs in the [design doc](https://github.com/fusor/ansible-playbook-bundle/blob/master/docs/design.md)
+  
+  
+<a name="prep-dev"></a>
+## Prepare your development environment 
+Install APB tooling as documented in the main [README](https://github.com/cfchase/ansible-playbook-bundle/tree/docs#install)
+
+Create a local development environment with both a [Service Catalog](https://github.com/kubernetes-incubator/service-catalog) and [Ansible Service Broker](https://github.com/fusor/ansible-service-broker).  We will do this using [catasb](https://github.com/fusor/catasb), a collection of scripts which use Ansible to automate the set up of the cluster for you on a local host, ec2, or virtual machines.  For this tutorial we'll be assuming the locally hosted environment which is documented at [https://github.com/fusor/catasb/blob/master/local/README.md](https://github.com/fusor/catasb/blob/master/local/README.md).  After completing the set up, take note of the OpenShift cluster **host:port** output by the catasb Ansible scripts so you can login using the command line for the remainder of the tutorial.  It will look something like:
+
+```
+TASK [debug] *********************************************************************************************************************************
+ok: [localhost] => {
+    "changed": false, 
+    "msg": [
+        "Hostname: <catasb-host>", 
+        "Next steps:", 
+        "1) Visit https://apiserver-service-catalog.<catasb-host>", 
+        "2) Accept the certificate", 
+        "3) Visit https://<catasb-host>:<port> for the console", 
+        "OR", 
+        "For CLI access:", 
+        "oc login --insecure-skip-tls-verify <catasb-host>:<port> -u admin -p admin", 
+        ""
+    ]
+}
+```
+
+Test out your development environment using [apb-examples](https://github.com/fusor/apb-examples):
+
+```bash
+git clone git@github.com:fusor/apb-examples.git
+cd hello-world-apb
+apb prepare
+docker build -t <docker-user>/hello-world-apb .
+docker run -e "OPENSHIFT_TARGET=https://<catasb-host>:<port>" -e "OPENSHIFT_USER=admin" -e "OPENSHIFT_PASS=admin" <docker-user>/hello-world-apb provision
+```
+This should have deployed a hello-world-apb application to your openshift cluster.  Logging on to your cluster at https://<catasb-host>:<port>, you should be able to view the project and click on the deployed application.
```

Comment:
Let's think more positive, 'This should have deployed...' change to 'This will have deployed ...'